apps/docs/src/guide/transactions/transaction-parameters.md

```diff
@@ -0,0 +1,28 @@
+# Transaction Parameters
+
+Transaction parameters allow you to configure various aspects of your blockchain transactions. The parameters are:
+
+1. Gas Price - The price you're willing to pay for each unit of [gas](https://docs.fuel.network/docs/intro/glossary/#gas) consumed during the transaction execution.
+1. Gas Limit - The maximum amount of gas you're willing to allow the transaction to consume. If the transaction execution requires more gas than the specified gas limit, the transaction will fail and any state changes made during the execution will be reverted.
+1. Maturity - The number of blocks that must pass before the transaction can be included in a block. This is useful for time-sensitive transactions, such as those involving time-locked assets.
+1. Max Fee - The maximum amount you're willing to pay for the transaction.
```

Comment:
The difference between the max fee and the gas limit is a bit unclear to me. Can you have another go at this?

Reply:
I've had a go but it is tricky. Gas limit is the total gas usage limit. And max fee is like gas used * gas price limit. Not sure if there's a better way of describing that?

https://github.com/FuelLabs/fuels-ts/pull/1961/commits/6daff848b53832ef7d88350fe850155fbf5fd5f2

Reply:
Can you just maybe add this -> `gas used * gas price limit` to the description too? That would help

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1961/commits/20871111460a759c3f845bb17eca973851defe6a